css/main.css

```diff
@@ -325,3 +325,73 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  background: #f4f5f7;
+  border-top-left-radius: 50px;
+  padding: 24px;
+  padding-top: 20%;
+  padding-bottom: 20%;
+  gap: 10%;
+  height: auto;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-me h2 {
+  color: #a04300;
+  font-weight: 600;
+  text-align: center;
+  font-size: 40px;
+}
```

Comment:
- [ ] Please from the project requirement you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

| Figma Design  |Your Design |
| :---:   | :---: | 
| ![Screenshot from 2022-10-22 00-57-37](https://user-images.githubusercontent.com/92303891/197309431-4034973b-0f9f-49e8-82ce-9a633e051598.png) | ![Screenshot from 2022-10-22 00-58-36](https://user-images.githubusercontent.com/92303891/197309474-aa3244c4-fde4-41b5-8dbf-40b419453691.png) |